backend/module/resolver/resolver.go

```diff
@@ -177,6 +186,52 @@ func (r *resolverAPI) GetObjectSchemas(ctx context.Context, req *resolverv1.GetO
 	}, nil
 }
 
+func (r *resolverAPI) Autocomplete(ctx context.Context, req *resolverv1.AutocompleteRequest) (*resolverv1.AutocompleteResponse, error) {
+	results := []*resolverv1.AutocompleteResponse_AutocompleteResult{}
+
+	limit := resolver.DefaultAutocompleteLimit
+	if req.Limit > 0 {
+		limit = int(req.Limit)
+	}
+
+	// Iterate through all of the available resolvers & schemas to find the one requested
+	// If that schema exists then we call the associated autocomplete function for that resolver
+	for _, res := range resolver.Registry {
+		resSchema := res.Schemas()
+		if _, ok := resSchema[req.Want]; ok {
+			resolverResults, err := res.Autocomplete(ctx, req.Want, req.Search, req.Limit)
+			if err != nil {
+				return nil, err
+			}
+
+			// Append results from all resolver up to the limit.
+			// Future enhancements here will try to surface the most relevant results
+			// from all resolvers instead of this basic approach
+			appendAutocompleteResultsToLimit(&results, resolverResults, limit)
+			if len(results) >= resolver.DefaultAutocompleteLimit {
```

Comment:
Shouldn't this use the limit?

Reply:
good catch i literally wrote the code for this above and did not use it.